app/models/imo_authorization.rb

```diff
@@ -0,0 +1,12 @@
+class ImoAuthorization < ApplicationRecord
```

Comment:
Note that we won't have history here of past invites if we are just using one record per patient.

Maybe that is fine, but we should be very careful we aren't spamming users with multiple invites over time due to bugs or errors in our logic. With just the one record, it will make it a bit harder to track that.

Reply:
Yeah, I figured this question might come up. I felt like having a record per invitation was over-engineering. I'd rather just add some logging like `sending invitation to #{patient_id}` than keep a stack of records just for debugging purposes. Maybe I'll regret that, but I would like to keep the design very slim here if possible.